.buildkite/pipelines/fleet/packages_daily.yml

```diff
@@ -25,3 +25,7 @@ steps:
     command: .buildkite/scripts/lifecycle/post_build.sh
     agents:
       queue: kibana-default
+
+notify:
+  - slack: "#fleet-notifications"
```

Comment:
This requires manual setup, can we swap the channel in the kibana-buildkite repository instead?

Reply:
Yeah the bot is sending the request out but there's a permissions issue.  Not sure if the channel is restricted, I can't view the settings.  If #fleet-notifications is open it shouldn't have any issues.

Reply:
yeah I think the permissions are restricted on #fleet-team channel. I'll update the channel in the kibana-buildkite repo.

Reply:
https://github.com/elastic/kibana-buildkite/pull/165